app/workflows/initiate_outbound_call.rb

```diff
@@ -0,0 +1,85 @@
+class InitiateOutboundCall < ApplicationWorkflow
+  class Error < StandardError; end
+
+  attr_reader :phone_call, :call_service_client
+
+  def initialize(phone_call, options = {})
+    @phone_call = phone_call
+    @call_service_client = options.fetch(:call_service_client) { CallService::Client.new }
+  end
+
+  def call
+    return unless phone_call.status.in?(%w[queued initiating])
+    return reschedule unless initiate!
+
+    remote_id = create_remote_call
+
+    mark_as_initiated!(remote_id)
```

Comment:
Should we just call `mark_as_initiated!(remote_id)` inside the `create_remote_call`? So we don't need the return value.